sources/src/AuthPlugin.ts

```diff
@@ -32,9 +32,14 @@ export class AuthPlugin {
 
     public initOnce: (authOptions: AuthOptions) => Promise<void> = once(async (authOptions) => {
         this.authOptions = authOptions;
-        const authenticated: boolean = this.guest.init(this.authOptions) || (await this.keycloak.init(this.authOptions));
+        const guestAuthenticated = this.guest.init(this.authOptions);
+        const keyCloakAuthenticated = await this.keycloak.init(this.authOptions);
```

Comment:
@khoanguyen102 i think we should not init the keycloak module when we have guest module.
in the old solution we also only initialized one module.

Reply:
Thanks @agentschmitt I have updated it